rust/scripts/build-release.sh

```diff
@@ -72,12 +72,20 @@ main() {
         find . -type f -name "libfilcrypto.a"
     fi
 
-    # generate filcrypto.h
-    # Check if FVM is in the build features - if so, include it in header generation
+    local __has_no_default_features=0
+    for arg in "${@:2}"; do
+        if [[ "$arg" == "--no-default-features" ]]; then
+            __has_no_default_features=1
+            break
+        fi
+    done
+
+    # If --no-default-features is set and FFI_DISABLE_FVM is not set to 1, add fvm
     local __header_features="c-headers"
-    if echo "${@:2}" | grep -q "fvm"; then
+    if [[ "${__has_no_default_features}" -eq 1 ]] && [[ "${FFI_DISABLE_FVM}" != "1" ]]; then
```

Comment:
```suggestion
    if [[ "${__has_no_default_features}" -eq 1 ]] || [[ -z "${FFI_DISABLE_FVM}" ]]; then
```

I think the `&&` is why CI is failing, `--no-default-features` is just an edge  case to handle, not a requirement. Also the `-z` added here cause you used it in install_filcrypto so we're no longer caring what value it's set to there, we should make it the same (I don't care either way as long as they are consistent).

Reply:
actually no, `--no-default-features` comes in here for header generation, but I think that oculd be dropped entirely, although your latest change has even more diff in here so I'll resolve this and look again